### PR TITLE
feat: add individual collection sheet 

### PR DESCRIPTION
**Pull request by pearlMat**
## Description
 individual collection sheet component completed. it generates a collection sheet component.

Related issues and discussion
#255 
## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [✅ ]  If you have multiple commits please combine them into one commit by squashing them.

- [✅] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.
cs
ics2

### CODE REVIEW

File: src/app/centers/individual-collection-sheet/individual-collection-sheet.component.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          </mat-form-field>
+          <!--End of form fields-->
+        </div>
+      </mat-card-content>
+      <!-- Generate collection sheet button -->
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="start" fxLayoutGap="5px">
+        <button mat-raised-button color="primary" [routerLink]="['collection-sheet']">
+          Collection Sheet
+        </button>
+      </mat-card-actions>
```
Comment (karantakalkar) [inline src/app/centers/individual-collection-sheet/individual-collection-sheet.component.html]: delineate here as well

File: src/app/organization/organization-routing.module.ts [hunk 2]
```diff
@@ -228,7 +228,7 @@ const routes: Routes = [
           resolve: {
             workingDays: WorkingDaysResolver
           }
-        }
+        },
```
Comment (karantakalkar) [inline src/app/organization/organization-routing.module.ts]:
```suggestion
        }
```

File: src/app/organization/organization.module.ts [hunk 3]
```diff
@@ -27,6 +27,7 @@ import { CreateEmployeeComponent } from './employees/create-employee/create-empl
 import { CreatePaymentTypeComponent } from './payment-types/create-payment-type/create-payment-type.component';
 import { ViewEmployeeComponent } from './employees/view-employee/view-employee.component';
 
+
```
Comment (karantakalkar) [inline src/app/organization/organization.module.ts]:
```suggestion
```

File: src/app/organization/organization.service.ts [hunk 4]
```diff
@@ -211,5 +211,4 @@ export class OrganizationService {
   updateWorkingDays(workingDays: any): Observable<any> {
     return this.http.put('/workingdays', workingDays);
   }
-
```
Comment (karantakalkar) [inline src/app/organization/organization.service.ts]: remove these changes

File: src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * Individual collection sheet component.
+ */
+@Component({
+  selector: 'mifosx-individual-collection-sheet',
+  templateUrl: './individual-collection-sheet.component.html',
+  styleUrls: ['./individual-collection-sheet.component.scss']
+})
+export class IndividualCollectionSheetComponent implements OnInit {
+    /** Collection sheet form. */
```
Comment (karantakalkar) [inline src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts]: variables are not aligned, indent properly

File: src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      repaymentDate: ['', Validators.required],
+      staff: ['']
+    });
+  }
+
+  ngOnInit() {
+    this.createCollectionSheetForm();
+  }
+
+  submit() {
```
Comment (karantakalkar) [inline src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts]: why have you not connected the component to API?

**Comment by BLasan** [comment]
Please put the issue number as #255 . Remove #679

File: src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts (hunk 6 above)
Comment (pearlMat) [reply src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts]: I do not have access to the data that populates the tables yet. I have used dummy data to fill the tables for now. Should i use dummy data to connect to the API?

File: src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts (hunk 5 above)
Comment (pearlMat) [reply src/app/centers/individual-collection-sheet/individual-collection-sheet.component.ts]: Done

File: src/app/organization/organization.service.ts (hunk 4 above)
Comment (pearlMat) [reply src/app/organization/organization.service.ts]: Done

File: src/app/centers/centers.module.ts [hunk 7]
```diff
@@ -5,10 +5,17 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { CoreModule } from '../core/core.module';
 import { MatTableModule } from '@angular/material/table';
 import { FlexLayoutModule } from '@angular/flex-layout';
-
+import { MatInputModule } from '@angular/material/input';
```
[the remark this answers is not in the document]
Comment (pearlMat) [reply src/app/centers/centers.module.ts]: Done all suggestions.

**Comment by karantakalkar** [comment]
@pearlMat can you resolve all conversations, please?

**Comment by pearlMat** [comment]
@karantakalkar i have done the fixes. i have not been able to connect to the api as i dont understand angularjs..i know it wont take long to pick up the basics..time is my problem.

**Comment by karantakalkar** [comment]
@pearlMat No issues, just let us know if you won't be able to complete the PR, so that other contributors can work on it.

**Comment by karantakalkar** [comment]
Closing due to inconsistency and inactivity

**Pull request closed without merging**